getFeatureFlagById/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"log"
+	"net/http"
+)
+
+func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	id, found := req.PathParameters["flagId"]
+
+	if !found {
+		log.Println("flagId not passed")
+		utils.ClientError(http.StatusBadRequest, "Flag ID not passed in request")
```

Comment:
Please add `return`.
Also, I think this case will never occur since if `flagId` path param is not passed then the path mentioned in `template.yaml` will not match and give an error before invoking the function

Reply:
Not sure, I think this lambda function will not even be invoked if the `flagId` is missing.

Reply:
Agreed, will remove once verification. @shubham-y

Reply:
We get missing authentication token.

Reply:
Yes but with the `found`1 check also we get the same response